embrace-android-delivery/src/main/kotlin/io/embrace/android/embracesdk/internal/delivery/execution/OkHttpRequestExecutionService.kt

```diff
@@ -81,10 +82,9 @@ class OkHttpRequestExecutionService(
     }
 
     private fun generateRequestBody(payloadStream: () -> InputStream) = object : RequestBody() {
-        override fun contentType() = null
+        override fun contentType() = "application/json".toMediaType()
```

Comment:
Lets create a constant for this and reuse it every time so we don't create a new object every time this is called?